sdk/java-sdk-spring/src/main/scala/kalix/spring/impl/KalixSpringApplication.scala

```diff
@@ -424,26 +428,45 @@ case class KalixSpringApplication(applicationContext: ApplicationContext, config
     ReflectiveEventSourcedEntityProvider.of(
       clz,
       messageCodec,
-      context => wiredInstance(clz) { case p if p == classOf[EventSourcedEntityContext] => context })
+      context =>
+        wiredInstance(clz) {
+          case p if p == classOf[EventSourcedEntityContext] => context
+          // we wire the Akka Config instance as it's the final one (see KalixRunner.prepareConfig)
+          case p if p == classOf[Config] => context.materializer().system.settings.config
+        })
 
   private def valueEntityProvider[S, VE <: ValueEntity[S]](clz: Class[VE]): ValueEntityProvider[S, VE] =
     ReflectiveValueEntityProvider.of(
       clz,
       messageCodec,
-      context => wiredInstance(clz) { case p if p == classOf[ValueEntityContext] => context })
+      context =>
+        wiredInstance(clz) {
+          case p if p == classOf[ValueEntityContext] => context
+          // we wire the Akka Config instance as it's the final one (see KalixRunner.prepareConfig)
+          case p if p == classOf[Config] => context.materializer().system.settings.config
```

Comment:
I think this line can be moved to `wiredInstance`. This way, we will avoid code duplication.

Reply:
Of course! 

I will clean this up.